src/kontrol/prove.py

```diff
@@ -480,6 +480,10 @@ def _method_to_cfg(
         init_node = cfg.create_node(init_cterm)
         new_node_ids = [init_node.id]
         init_node_id = init_node.id
+        if not method.is_testfail and not method.is_test:
+            _LOGGER.warning(
```

Comment:
Let's make it an INFO-level log (as per this [comment](https://github.com/runtimeverification/kontrol/issues/386#issuecomment-1957198515)):
```suggestion
            _LOGGER.info(
```

Reply:
But then it will only be printed with `--verbose`.

Reply:
Oh right! I'm ok with it being a warning (or even a `print` statement) to make sure that people see it — there already were several occasions on which users were confused that such non-test functions weren't failing as expected. 

@ehildenb but if you find it too strong, we can leave it like that.